Matveibos/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+ENTER_STRING = 'enter string'
+ERROR = 'String can not be blank!'
+EXIT = 'exit!'
+RESULT = 'Middle character:'
+def get_middle(str)
+  slen = (str.length - 1) / 2.0
+  str[slen.floor..slen.ceil]
+end
+
+def run_cli
+  loop do
+    puts ENTER_STRING
+    st = gets.chomp
+    break if st == EXIT
+
+    st.empty? ? (puts 'String can not be blank!') : (puts RESULT + get_middle(st))
```

Comment:
You just moved `'String can not be blank!'` to the `ERROR` constant, so lets use it there
Also prefer string interpolation, rather then concatenation, at least in this case: `RESULT + get_middle(st)`

Reply:
please excuse my inattention)
Ok!